src/providers/discord.ts

```diff
@@ -255,16 +390,19 @@ async function snapshotCommandHandler(interaction, commandType) {
     if (!space)
       return interaction.reply(`Space not found: ${inlineCode(spaceId)}`);
 
+    const events = await getEventsConfigured(interaction.guildId);
+
     const subscription = [
       interaction.guildId,
       channelId,
       spaceId,
       mention || '',
+      events ? JSON.stringify(events) : undefined,
```

Comment:
The SQL insert fails with `Column 'events' cannot be null`. We should keep the fallback value here